docs/applications/edit-application.md

```diff
@@ -0,0 +1,26 @@
+---
+draft: true
```

Comment:
why draft?

Reply:
This post's goal (see Jira issue) is to teach how to update an application repository, an option currently not available in the web app.

Reply:
If we just remove the steps below, and remove it from draft, the post will already be useful and might encourage other users to contribute to improve it, or am I missing something?

```md
1. Select a connected `Source Provider` account
    - If you want to use a different one, click `Connect a new source provider account` and follow the prompts
1. Use the drop-down menu to choose the repository
```

Reply:
But anyway, if you wanna keep it in draft, that's fine. I'm approving anyway.